pkg/assert/service.go

```diff
@@ -0,0 +1,40 @@
+package assert
+
+import (
+	"net"
+	"strings"
+
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/gomega"
+)
+
+// CheckServiceSpecContainsSubstring retrives service specs and validates expected substring
+func CheckServiceSpecContainsSubstring(svc, expected string) {
+	cmd := "kubectl get svc "+svc+" -n default"+
+		" -o jsonpath='{range .items[*]}{.spec}' --kubeconfig=" + shared.KubeConfigFile
+	res, err := shared.RunCommandHost(cmd)
+	Expect(err).NotTo(HaveOccurred(), err)
+	Expect(res).To(ContainSubstring(expected))
+}
+
+// ValidateClusterIPsBySVC retrives cluster IPs by svc and validates them in CIDR Range  
+func ValidateClusterIPsBySVC(svc string, expected []string) {
+	cmd := "kubectl get svc "+svc+
+		` -o jsonpath='{.spec.clusterIPs[*]}' --kubeconfig=` + shared.KubeConfigFile
+	Eventually(func() error {
+		res, _ := shared.RunCommandHost(cmd)
+		ips := strings.Split(res, " ")
+		Expect(len(ips)).ShouldNot(BeZero())
+		Expect(len(expected)).ShouldNot(BeZero())
```

Comment:
Same as what Fran commented before [here](https://github.com/rancher/distros-test-framework/pull/66/files#r1430130617). This can be in line number 23, say.